Test.Integration.Net7/EventsAPI/Account.cs

```diff
@@ -0,0 +1,199 @@
+                    name = "Bill Jones",
+                    phone = "1-415-555-6041",
+                    address_1 = "2100 Main Street",
+                    address_2 = "Apt 3B",
+                    city = "New London",
+                    region = "New Hampshire",
+                    country = "US",
+                    zipcode = "03257"
+                },
+                shipping_address = new Address()
+                {
+                    name = "Bill Jones",
+                    phone = "1-415-555-6041",
+                    address_1 = "2100 Main Street",
+                    address_2 = "Apt 3B",
+                    city = "New London",
+                    region = "New Hampshire",
+                    country = "US",
+                    zipcode = "03257"
+                },
+                promotions = new ObservableCollection<Promotion>()
+                {
+                    new Promotion()
+                    {
+                        promotion_id = "FirstTimeBuyer",
+                        status = "$success",
+                        description = "$5 off",
+                        discount = new Discount()
+                        {
+                            amount = 5000000,
+                            currency_code = "USD",
+                            minimum_purchase_amount = 25000000
+                        }
+                    }
+                },
+                social_sign_on_type = "$twitter",
+                browser = new Browser
+                {
+                    user_agent = "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_12_3) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/56.0.2924.87 Safari/537.36",
+                    accept_language = "en-US",
+                    content_language = "en-GB"
+                },
+                account_types = new ObservableCollection<string>() { "merchant", "premium" },
+                brand_name = "sift",
+                site_country = "US",
+                site_domain = "sift.com",
+                merchant_profile = new MerchantProfile()
+                {
+                    merchant_id = environmentVariable.merchant_id,
+                    merchant_category_code = "1234",
+                    merchant_name = "Dream Company",
+                    merchant_address = new Address()
+                    {
+                        phone = "1-415-555-6040",
+                        address_1 = "2100 Main Street",
+                        address_2 = "Apt 3B",
+                        city = "New London",
+                        region = "New Hampshire",
+                        country = "US",
+                        zipcode = "03257"
+
+                    }
+                }
+            };
+            EventRequest eventRequest = new EventRequest()
+            {
+                Event = createAccount
+            };
+            EventResponse res = sift.SendAsync(eventRequest).Result;
+            Assert.Equal("OK", res.ErrorMessage);
```

Comment:
it is better to check not `ErrorMessage`, but `Status`, and successful one is `0`

Reply:
@binish-exalture, pls take a look